src/components/Input.tsx

```diff
@@ -75,11 +79,13 @@ class Input extends React.PureComponent<InputProps, InputState> {
 
     const Input_ = textArea ? "textarea" : "input";
 
-    const inputClassName = cx(inputStyle, {
-      _pebble_input_disabled: disabled,
-      _pebble_input_read_only: readOnly,
-      _pebble_input_textarea: textArea
-    });
+    const _inputClassName = cx(
+      inputStyle,
+      !!disabled && inputDisabledStyle,
+      !!readOnly && inputReadOnlyStyle,
+      !!textArea && inputTextAreaStyle,
```

Comment:
use the recommended syntax.

```js
const _inputClassName = cx(inputStyle, {
      inputDisabledStyle : disabled,
      inputReadOnlyStyle: readOnly,
      inputTextAreaStyle: textArea
}
```

Reply:
That will not work, as `inputDisabledStyle` is a variable not a string

Reply:
updated

Reply:
done